webapp/views/partials/home.jade

```diff
@@ -0,0 +1,8 @@
+div(class='container')
+    h1 {{userName}}
+
+    div(class='div-center')
```

Comment:
does ```text-center``` fit your needs here? It's used in ```main.jade``` and it seems to keep everything centered. If so, you should also remove the css for ```div-center``` in the css file.